Update getting started guide (#5162)
tuist/tuist: projects/docs/docs/tutorial/get-started.md

To avoid that, Tuist is self-contained and comes with `tuistenv` which ensures that the right version is used. It manages different versions in your environment and runs the version your project is pinned to. Thanks to that, we ensure that anyone in your team will use the same version of Tuist.  
In a more advanced section on the documentation, we'll see the power of `tuistenv`. For now, we'll keep things simple and just assume that we are running Tuist directly.

> ### Uninstall
> 
> To manage tuist versions etc it is recommended to use the `tuist` cli directly, like `tuist uninstall version`
> But if one wishes to remove tuist from the system completly it is recommended to use the uninstall script.
> Note: This will remove Tuist only if you have used the above installation steps
> 
> ```bash
> curl -Ls https://raw.githubusercontent.com/tuist/tuist/main/script/uninstall | bash
> ```

Reviewer: ```suggestion
#### Uninstall

If you wish to remove `tuist` from the system, you can use the uninstall script:

```bash
curl -Ls https://raw.githubusercontent.com/tuist/tuist/main/script/uninstall | bash
```
```
